app/controllers/carts_controller.rb

```diff
@@ -1,14 +1,25 @@
 class CartsController < ApplicationController
   def create
+    if session[:cart]
+      if session[:cart][params[:item_id]]
+        session[:cart][params[:item_id]]  += 1
+      else
+        session[:cart][params[:item_id]] = 1
+      end
+    else
+      session[:cart] = {params[:item_id] => 1}
+    end
+    @cart_items = session[:cart]
 
+    flash[:notice] = 'Added '  + Item.find_by_dao(params[:item_id]).name
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.